src/java.desktop/share/native/libharfbuzz/hb-ot-shape-complex-use-machine.hh

```diff
@@ -375,7 +375,7 @@ hb_iter_with_fallback_t<machine_index_t<Iter>,
 typename Iter::item_t>
 {
         machine_index_t (const Iter& it) : it (it) {}
-        machine_index_t (const machine_index_t& o) : it (o.it) {}
+        machine_index_t (const machine_index_t& o) : hb_iter_with_fallback_t<machine_index_t<Iter>, typename Iter::item_t>(o), it (o.it) {}
```

Comment:
is it relevant?  
I feel it belongs to a separated issue.

Reply:
Sorry, forgot to mention. Unrelated harfbuzz build fix for gcc 8 and earlier. Will remove before the final push.

Reply:
> The API change looks good to me. I understand that this is prerequisite of your logdecoration change. make sense.
> 
> I don't understand your change in harfbuzz header.

Thanks for your review, Xin.